README.md

```diff
@@ -53,9 +49,9 @@ ssl_crt: |
   -----END CERTIFICATE-----
 ```
 
-* Install or update ansible playbooks using [librarian-ansible](https://github.com/bcoe/librarian-ansible):
+* Install or update ansible playbooks using [ansible-galaxy](http://docs.ansible.com/galaxy.html):
 ```
-bundle exec librarian-ansible install
+ansible-galaxy install -r requirements.yml
```

Comment:
Because they're using `--vault-password-file=password.txt` or something else?

I'm inclined to think that we should either fix that or refer people to the contents of the `Makefile`, because if we duplicate content in the `README` then one of them is likely to fall out-of-date.